models/models.go

```diff
@@ -0,0 +1,84 @@
+package models
+
+import (
+	"time"
+)
+
+// DBModel contains basic primary key attribute for most entities
+type DBModel struct {
+	ID        uint       `gorm:"primary_key"`
+	CreatedAt time.Time  `json:"-"`
+	UpdatedAt time.Time  `json:"-"`
+	DeletedAt *time.Time `json:"-"`
+}
+
+// Address represents one address
+type Address struct {
+	DBModel
+	Line1   string
+	Line2   string
+	Pincode string
+}
+
+// User represents entity having all attributes required to represent an app user
+type User struct {
+	DBModel
+	FName    string
+	LName    string
+	UName    string
+	Email    string
+	Vehicles string
```

Comment:
What does Vehicles represent! is it only one vehicle or does it include multiple vehicles